cmd/gitops/beta/run/cmd.go

```diff
@@ -525,6 +527,72 @@ func betaRunCommandRunE(opts *config.Options) func(*cobra.Command, []string) err
 			return err
 		}
 
+		// run bootstrap wizard only if Flux was not installed
+		if fluxVersion != "" {
```

Comment:
:no_entry: Final thought: it's Release Week, and the follow up tickets won't be done before we want to ship. Could we just add a quick check here that, say, the `GITOPS_RUN_BOOTSTRAP` env var is set, and if not also return nil?

Reply:
Sure, not a problem.

Should I just check for `os.Getenv("GITOPS_RUN_BOOTSTRAP")` or do I also need a call to `osSetenv` somewhere?

Reply:
No setenv, just check.